Riot/Modules/Contacts/DataSources/ContactsDataSource.m

```diff
@@ -557,7 +558,8 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
         {
             contact = filteredMatrixContacts[indexPath.row];
             
-            showMatrixIdInDisplayName = self.forceMatrixIdInDisplayName ? YES : [isMultiUseNameByDisplayName[contact.displayName] isEqualToNumber:@(YES)];
+            // Tchap: Force to hide MatrixID in DisplayName
+//            showMatrixIdInDisplayName = self.forceMatrixIdInDisplayName ? YES : [isMultiUseNameByDisplayName[contact.displayName] isEqualToNumber:@(YES)];
```

Comment:
I would keep the Element-iOS code here, normally the case where multiple accounts have the same display name is limited in Tchap. We can keep the code as soon as `forceMatrixIdInDisplayName` is `false`